src/giftCards/GiftCardsList/providers/GiftCardListDialogsProvider/hooks/useGiftCardListSort.ts

```diff
@@ -0,0 +1,19 @@
+import { useContext } from "react";
+
+import {
+  GiftCardsListConsumerProps,
+  GiftCardsListContext
+} from "../../GiftCardListProvider";
+
+type GiftCardSortListProps = Pick<
+  GiftCardsListConsumerProps,
+  "onSort" | "sort"
+>;
+
+const useGiftCardListSort = (): GiftCardSortListProps => {
+  const { onSort, sort } = useContext(GiftCardsListContext);
+
+  return { onSort, sort };
+};
```

Comment:
Don't we already have wrapper around `useContext(GiftCardsListContext)`?

Reply:
Technically yes but I don't think exporting `onSort, sort` from these hooks (`useGiftCardList` and `useGiftCardListBulkActions`) is the best solution

Reply:
I don't see why, can you elaborate?

Reply:
@dominik-zeglen I just didn't think that adding `sort, onSort` to the main gift card hook would look clean (one hook do to it all) but I think you're right and it's fine.